stage0/src/fw_cfg.rs

```diff
@@ -69,6 +69,16 @@ impl DirEntry {
     }
 }
 
+impl core::fmt::Debug for DirEntry {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("DirEntry")
+            .field("name", &self.name())
+            .field("size", &self.size())
+            .field("select", &self.selector())
+            .finish()
+    }
+}
+
 pub struct FwCfg {
```

Comment:
pre-existing: could you please add a comment what "Fw" stands for?

Reply:
Done.